usage/plugins/org.jboss.tools.usage/src/org/jboss/tools/usage/internal/environment/eclipse/AbstractEclipseEnvironment.java

```diff
@@ -161,43 +152,40 @@ private String createIdentifier() {
 	}
 
 	@Override
-	public abstract String getKeyword();
```

Comment:
Because this method is defined in a super interface and we are here in an abstract class so no need to re declare it